### PR TITLE
Fixed ProgressBar.log_message.

### LINKED ISSUE #767: log_message() method fails `desc` is passed to contrib tqdm
## 🐛 Bug description

If I pass a `desc` argument when instantiating a `ignite.contrib.handlers.ProgressBar` the calls to its `log_message()` method fail with this exception:

```
TypeError: write() got an unexpected keyword argument 'desc'
```

## Environment

 - PyTorch Version: 1.3.1
 - Ignite Version: 0.3.0
 - OS: Linux
 - How you installed Ignite (`conda`, `pip`, source): Conda
 - Python version: 3.7
 - Any other relevant information:

### PR DESCRIPTION
**Pull request by maxfrei750**
Fixes #767.

Check list:
* [x] New tests are added (if a new feature is added)
* [ ] New doc strings: description and/or example code are in RST format
* [ ] Documentation is updated (if required)

### CODE REVIEW

**Comment by maxfrei750** [comment]
I added a new test to fix the coverage. However, on my machine nothing was written to the test output file. I cannot see why not, so I wanted to give it a try on the test system.

**Comment by maxfrei750** [comment]
@vfdev-5 Ok, I found the error. The test output file was just written after it had been closed at program termination.

File: ignite/contrib/handlers/tqdm_logger.py [hunk 1]
```diff
@@ -145,7 +145,13 @@ def log_message(self, message):
             message (str): string you wish to log.
         """
         from tqdm import tqdm
-        tqdm.write(message, **self.tqdm_kwargs)
+
+        if "file" in self.tqdm_kwargs:
```
Comment (justusschock) [inline ignite/contrib/handlers/tqdm_logger.py]: can we replace this part with `tqdm.write(message, file=self.tqdm_kwargs.get('file', None))`?

Comment (maxfrei750) [reply ignite/contrib/handlers/tqdm_logger.py]: Sure! That's way more elegant.

**Comment by vfdev-5** [comment]
Seems like tests on pytorch-nightly are not using nightly builds: https://travis-ci.org/pytorch/ignite/jobs/647336703#L514
This may fail CI etc...

**Comment by maxfrei750** [comment]
@vfdev-5 Sorry, but I don't seem to understand. Is there anything that can be done or will the problem just take some time to resolve?

**Comment by vfdev-5** [comment]
@maxfrei750 sorry, I just wanted to mention that there is a problem with CI and it does not depend on your PR. So, yes, probably it would take time to resolve it...

**Pull request merged**